aws/table_aws_eks_node_group.go

```diff
@@ -0,0 +1,257 @@
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEksNodeGroup,
+			},
+			{
+				Name:        "instance_types",
+				Description: "If the node group wasn't deployed with a launch template, then this is the instance type that is associated with the node group.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEksNodeGroup,
+			},
+			{
+				Name:        "subnets",
+				Description: "The subnets that were specified for the Auto Scaling group that is associated with node group.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEksNodeGroup,
+			},
+			{
+				Name:        "remote_access",
+				Description: "If the node group wasn't deployed with a launch template, then this is the remote access configuration that is associated with the node group.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEksNodeGroup,
+			},
+			{
+				Name:        "ami_type",
+				Description: "If the node group was deployed using a launch template with a custom AMI, then this is CUSTOM. For node groups that weren't deployed using a launch template, this is the AMI type that was specified in the node group configuration.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getEksNodeGroup,
+			},
+			{
+				Name:        "node_role",
+				Description: "The IAM role associated with your node group.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getEksNodeGroup,
+			},
+			{
+				Name:        "labels",
+				Description: "If the node group wasn't deployed with a launch template, then this is the remote access configuration that is associated with the node group.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEksNodeGroup,
+			},
+			{
+				Name:        "resources",
+				Description: "The resources associated with the node group, such as Auto Scaling groups and security groups for remote access.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEksNodeGroup,
+			},
+			{
+				Name:        "disk_size",
+				Description: "If the node group wasn't deployed with a launch template, then this is the disk size in the node group configuration. If the node group was deployed with a launch template, then this is null.",
+				Type:        proto.ColumnType_INT,
+				Hydrate:     getEksNodeGroup,
+			},
+			{
+				Name:        "health",
+				Description: "The health status of the node group.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEksNodeGroup,
+			},
+			{
+				Name:        "launch_template",
+				Description: "If a launch template was used to create the node group, then this is the launch template that was used.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEksNodeGroup,
+			},
+			{
+				Name:        "tags_src",
+				Description: "A list of tags attached to the node group.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEksNodeGroup,
+				Transform:   transform.FromField("tags"),
```

Comment:
If we are getting the tags in TagsMap format the why we are adding tags_src column . And also please keep the `tags` column in ` Standard columns` section